external_search.py

```diff
@@ -406,6 +407,14 @@ def query_works(self, query_string, filter=None, pagination=None,
                     i, result.title, result.author, result.meta['id'],
                     result.meta['score'] or 0, result.meta['shard']
                 )
+
+        # Convert the Search object into a list of hits.
+        results = [x for x in results]
```

Comment:
This introduces a tiny inefficiency because in most cases we iterate over the list twice, but the list isn't that big.